### PR TITLE
Fixes multiline markdown link text

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/audiobuffersourcenode/playbackrate/index.md [hunk 1]
```diff
@@ -42,9 +42,7 @@ decode an audio track, and put it into an {{domxref("AudioBufferSourceNode")}}.
 are provided to play and stop the audio playback, and a slider control is used to change
 the `playbackRate` property value on the fly.
 
-> **Note:** You can [run the example
-> live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view
-> the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).) Play the song and alter the playback rate for some fun results.
+> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).) Play the song and alter the playback rate for some fun results.
```
Comment (teoli2003) [inline files/en-us/web/api/audiobuffersourcenode/playbackrate/index.md:45]:
```suggestion
> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data)). Play the song and alter the playback rate for some fun results.
```

File: files/en-us/web/api/baseaudiocontext/createscriptprocessor/index.md [hunk 2]
```diff
@@ -70,8 +70,7 @@ buffer, and each sample in each channel, and add a small amount of white noise,
 setting that result to be the output sample in each case.
 
 > **Note:** For a full working example, see our [script-processor-node](https://mdn.github.io/webaudio-examples/script-processor-node/)
-> GitHub repo (also view the [source
-> code](https://github.com/mdn/webaudio-examples/blob/master/script-processor-node/index.html).)
+> GitHub repo (also view the [source code](https://github.com/mdn/webaudio-examples/blob/master/script-processor-node/index.html).)
```
Comment (teoli2003) [inline files/en-us/web/api/baseaudiocontext/createscriptprocessor/index.md:73]:
```suggestion
> GitHub repo. (You can also access the [source code](https://github.com/mdn/webaudio-examples/blob/master/script-processor-node/index.html).)
```

File: files/en-us/web/api/baseaudiocontext/decodeaudiodata/index.md [hunk 3]
```diff
@@ -77,9 +77,7 @@ The buttons in the example run `getData()` to load the track and start it
 playing, and stop it playing, respectively. When the `stop()` method is
 called on the source, the source is cleared out.
 
-> **Note:** You can [run the example
-> live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view
-> the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).)
+> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).)
```
Comment (teoli2003) [inline files/en-us/web/api/baseaudiocontext/decodeaudiodata/index.md:80]:
```suggestion
> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and acces the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).
```

File: files/en-us/web/api/eventsource/close/index.md [hunk 4]
```diff
@@ -44,8 +44,7 @@ button.onclick = function() {
 }
 ```
 
-> **Note:** You can find a full example on GitHub — see [Simple SSE
-> demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
+> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
```
Comment (teoli2003) [inline files/en-us/web/api/eventsource/close/index.md:47]:
```suggestion
> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP](https://github.com/mdn/dom-examples/tree/master/server-sent-events).
```

File: files/en-us/web/api/eventsource/eventsource/index.md [hunk 5]
```diff
@@ -49,8 +49,7 @@ evtSource.onmessage = function(e) {
 }
 ```
 
-> **Note:** You can find a full example on GitHub — see [Simple SSE
-> demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
+> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
```
Comment (teoli2003) [inline files/en-us/web/api/eventsource/eventsource/index.md:52]:
```suggestion
> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP](https://github.com/mdn/dom-examples/tree/master/server-sent-events).
```

File: files/en-us/web/api/eventsource/url/index.md [hunk 6]
```diff
@@ -27,8 +27,7 @@ var evtSource = new EventSource('sse.php');
 console.log(evtSource.url);
 ```
 
-> **Note:** You can find a full example on GitHub — see [Simple SSE
-> demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
+> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
```
Comment (teoli2003) [inline files/en-us/web/api/eventsource/url/index.md:30]:
```suggestion
> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP](https://github.com/mdn/dom-examples/tree/master/server-sent-events).
```

File: files/en-us/web/api/eventsource/withcredentials/index.md [hunk 7]
```diff
@@ -29,8 +29,7 @@ var evtSource = new EventSource('sse.php');
 console.log(evtSource.withCredentials);
 ```
 
-> **Note:** You can find a full example on GitHub — see [Simple SSE
-> demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
+> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
```
Comment (teoli2003) [inline files/en-us/web/api/eventsource/withcredentials/index.md:32]:
```suggestion
> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP](https://github.com/mdn/dom-examples/tree/master/server-sent-events).
```

File: files/en-us/web/api/idbkeyrange/bound/index.md [hunk 8]
```diff
@@ -67,9 +67,7 @@ used `IDBKeyRange.bound("A", "F", true, true);`, then the range would not
 include `"A"` and `"F"`, only the values between them.
 
 > **Note:** For a more complete example allowing you to experiment with
-> key range, have a look at the idbkeyrange directory [in the
-> indexeddb-examples](https://github.com/mdn/indexeddb-examples/tree/master/idbkeyrange) repo ([view the example live
-> too](https://mdn.github.io/indexeddb-examples/idbkeyrange/).)
+> key range, have a look at the idbkeyrange directory [in the indexeddb-examples](https://github.com/mdn/indexeddb-examples/tree/master/idbkeyrange) repo ([view the example live too](https://mdn.github.io/indexeddb-examples/idbkeyrange/).)
```
Comment (teoli2003) [inline files/en-us/web/api/idbkeyrange/bound/index.md:70]:
```suggestion
> key range, have a look at the idbkeyrange directory in the [indexeddb-examples](https://github.com/mdn/indexeddb-examples/tree/master/idbkeyrange) repo. (Vies the example [live](https://mdn.github.io/indexeddb-examples/idbkeyrange/) too.
```

File: files/en-us/web/api/idbkeyrange/lower/index.md [hunk 9]
```diff
@@ -39,8 +39,7 @@ console, which should appear as "F".
 
 > **Note:** For a more complete example allowing you to experiment with
 > key range, have a look at our [IDBKeyRange-example](https://github.com/mdn/indexeddb-examples/tree/master/idbkeyrange) repo
```
Comment (teoli2003) [inline files/en-us/web/api/idbkeyrange/lower/index.md:41]:
```suggestion
> key range, have a look at our [IDBKeyRange-example](https://github.com/mdn/indexeddb-examples/tree/master/idbkeyrange) repo.
```

File: files/en-us/web/api/idbkeyrange/lower/index.md [hunk 10]
```diff
@@ -39,8 +39,7 @@ console, which should appear as "F".
...
-> ([view the example live
-> too](https://mdn.github.io/indexeddb-examples/idbkeyrange/).)
+> ([view the example live too](https://mdn.github.io/indexeddb-examples/idbkeyrange/).)
```
Comment (teoli2003) [inline files/en-us/web/api/idbkeyrange/lower/index.md:42]:
```suggestion
> (View the example [live](https://mdn.github.io/indexeddb-examples/idbkeyrange/) too.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Some nits.

**Comment by OnkarRuikar** [comment]
Thanks @teoli2003 !

File: files/en-us/web/api/baseaudiocontext/decodeaudiodata/index.md [hunk 11]
```diff
@@ -77,7 +77,7 @@ The buttons in the example run `getData()` to load the track and start it
 playing, and stop it playing, respectively. When the `stop()` method is
 called on the source, the source is cleared out.
 
-> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).)
+> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and acces the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).
```
Comment (teoli2003) [inline files/en-us/web/api/baseaudiocontext/decodeaudiodata/index.md:80]:
```suggestion
> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and access the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**